submissions/DarkSoulH/Hooli-style Popup/style.css

```diff
@@ -0,0 +1,359 @@
+    font-weight: 600;
+
+    color: #6f6f6f;
+
+    border-radius: 10px;
+    padding: 5px;
+}
+
+.menu_item_link:hover {
+    text-decoration: underline;
+}
+
+.menu_item_link:focus,
+.menu_item_link:focus-visible {
+    outline: none;
+
+    background-color: #b4e2f5;
+
+    box-shadow: 0px 0px 3px 1px #b1aeac;
+}
+
+.checkbox,
+.checkbox_sequel {
+    position: absolute;
+
+    opacity: 0;
+
+    cursor: pointer;
+
+    height: 0;
+    width: 0;
+}
+
+.popup_button {
+    border-radius: 50%;
+}
+
+.checkbox:checked ~ label .popup_button,
+.checkbox:focus ~ label .popup_button,
+.popup_button:hover {
+    background-color: #b4e2f5;
+
+    box-shadow: 0px 0px 3px 1px #b1aeac;
+}
+
+.popup_wrapper {
+    display: none;
+
+    position: absolute;
+
+    top: 44px;
+    right: -80px;
+
+    overflow: hidden;
+
+    width: 250px;
+    height: 320px;
+
+    padding: 0 4px;
+
+    background-color: #fff;
+
+    border-radius: 9px;
+    box-shadow: 0px 1px 2px 0px rgb(60 64 67 / 30%),
+                0px 2px 6px 2px rgb(60 64 67 / 15%);
+
+                z-index: 1;
+}
+
+.checkbox:checked ~ .popup_wrapper {
+    display:block;
+}
+
+.popup_content {
+    height: 320px;
+    overflow: auto;
+}
+
+.popup_content::-webkit-scrollbar {
+    width: 8px;
+}
+
+.popup_content::-webkit-scrollbar-thumb {
+    background-color: #e1e1e1;
+    border-radius: 9px;
+}
+
+.popup_content::-webkit-scrollbar-track {
+    margin: 5px;
+}
+
+.popup_list {
+    display: flex;
+    flex-wrap: wrap;
+
+    justify-content: center;
+    align-items: center;
+
+    padding-top: 10px;
+}
+
+.popup_list_item {
+    display: flex;
+
+    align-items: flex-end;
+    justify-content: center;
+
+    text-align: center;
+
+    margin-bottom: 10px;
+
+}
+
+.app_name {
+    display: block;
+
+    text-align: center;
+
+    padding-top: 10px;
+
+    font-size: 12px;
+    font-weight: 300;
+
+    color: black;
+}
+
+.app_name:hover {
+    text-decoration: underline;
+}
+
+.app_icon {
+    display: inline-block;
+
+    width: 70px;
+    height: fit-content;
+    
+    text-align: center;
+
+    padding: 5px 0;
+
+    margin-bottom: 12px;
+
+    border-radius: 3px;
+    border: 1px solid transparent;
+}
+
+.app_icon:hover {
+    background-color: rgba(106, 168, 255, 0.135);
+
+    cursor: pointer;
+}
+
+.app_icon:focus {
+    background-color: rgba(106, 168, 255, 0.135);
+
+    box-shadow: 0px 0px 3px 1px #b1aeac;
+
+    outline: none;
+}
+
+.app_icon:focus .app_name {
+    text-decoration: underline;
+
+    color: black;
+}
+
+.icon {
+    max-height: 35px;
+}
+
+.menu_item_bell {
+    border-radius: 50%;
+
+    padding: 3px;
+
+    font-family: unset;
+}
+
+.bell {
+    width: 20px;
+    height: 20px;
+}
+
+.menu_item_soft {
+    display: flex;
+    
+    align-items: center;
+    justify-content: center;
+
+    width: 25px;
+    height: 25px;
+
+    border-radius: 50%;
+
+    color: rgb(247, 234, 253);
+    background: #738f84;
+
+    transition: color .1s linear;
```

Comment:
To avoid reassigning properties use shortline assignment. [Read more](https://dev.to/clairecodes/css-transitions-for-multiple-properties-on-one-element-9o)